#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/programs--pluto--pluto.8-linebreak.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP:
## DP: Author: Jari Aalto <jari.aalto@cante.net>
## DP:
## DP:
## DP: (can't break line)

@DPATCH@

diff --git a/programs/pluto/pluto.8 b/programs/pluto/pluto.8
index 16523d6..20b700b 100644
--- a/programs/pluto/pluto.8
+++ b/programs/pluto/pluto.8
@@ -44,7 +44,7 @@ ipsec_pluto \- ipsec whack : IPsec IKE keying daemon and control interface
 .br

 .br
-[\-\-tunnel] [\-\-psk] [\-\-rsasig] [\-\-encrypt] [\-\-authenticate] [\-\-compress] [\-\-pfs] [\-\-pfsgroup\ [modp1024]\ |\ [modp1536]\ |\ [modp2048]\ |\ [modp3072]\ |\ [modp4096]\ |\ [modp6144]\ |\ [modp8192]] [\-\-disablearrivalcheck] [\-\-ikelifetime\ \fIseconds\fR] [\-\-ipseclifetime\ \fIseconds\fR] [\-\-rekeymargin\ \fIseconds\fR] [\-\-rekeyfuzz\ \fIpercentage\fR] [\-\-keyingtries\ \fIcount\fR] [\-\-esp\ \fIesp\-algos\fR] [\-\-dontrekey] [\-\-aggrmode] [\-\-modecfgpull] [[\-\-dpddelay\ \fIseconds\fR] | [\-\-dpdtimeout\ \fIseconds\fR]] [\-\-dpdaction\ [clear]\ |\ [hold]\ |\ [restart]] [\-\-forceencaps] [[\-\-initiateontraffic]\ |\ [\-\-pass]\ |\ [\-\-drop]\ |\ [\-\-reject]] [[\-\-failnone]\ |\ [\-\-failpass]\ |\ [\-\-faildrop]\ |\ [\-\-failreject]] [\-\-ctlbase\ \fIpath\fR] [\-\-optionsfrom\ \fIfilename\fR] [\-\-label\ \fIstring\fR]
+[\-\-tunnel] [\-\-psk] [\-\-rsasig] [\-\-encrypt] [\-\-authenticate] [\-\-compress] [\-\-pfs] [\-\-pfsgroup [modp1024] | [modp1536] | [modp2048] | [modp3072] | [modp4096] | [modp6144] | [modp8192]] [\-\-disablearrivalcheck] [\-\-ikelifetime \fIseconds\fR] [\-\-ipseclifetime \fIseconds\fR] [\-\-rekeymargin \fIseconds\fR] [\-\-rekeyfuzz \fIpercentage\fR] [\-\-keyingtries \fIcount\fR] [\-\-esp \fIesp\-algos\fR] [\-\-dontrekey] [\-\-aggrmode] [\-\-modecfgpull] [[\-\-dpddelay \fIseconds\fR] | [\-\-dpdtimeout \fIseconds\fR]] [\-\-dpdaction [clear] | [hold] | [restart]] [\-\-forceencaps] [[\-\-initiateontraffic] | [\-\-pass] | [\-\-drop] | [\-\-reject]] [[\-\-failnone] | [\-\-failpass] | [\-\-faildrop] | [\-\-failreject]] [\-\-ctlbase \fIpath\fR] [\-\-optionsfrom \fIfilename\fR] [\-\-label \fIstring\fR]
 .HP \w'\fBipsec\fR\ 'u
 \fBipsec\fR \fIwhack\fR \-\-keyid\ \fIid\fR [\-\-addkey] [\-\-pubkeyrsa\ \fIkey\fR] [\-\-ctlbase\ \fIpath\fR] [\-\-optionsfrom\ \fIfilename\fR] [\-\-label\ \fIstring\fR]
 .HP \w'\fBipsec\fR\ 'u
